google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -764,15 +775,33 @@ public Query orderBy(@Nonnull FieldPath fieldPath, @Nonnull Direction direction)
   }
 
   /**
-   * Creates and returns a new Query that's additionally limited to only return up to the specified
-   * number of documents.
+   * Creates and returns a new Query that only returns the first matching documents.
    *
    * @param limit The maximum number of items to return.
    * @return The created Query.
    */
   @Nonnull
   public Query limit(int limit) {
-    return new Query(firestore, options.toBuilder().setLimit(limit).build());
+    return new Query(
+        firestore, options.toBuilder().setLimit(limit).setLimitType(LimitType.First).build());
+  }
+
+  /**
+   * Creates and returns a new Query that only returns the last matching documents.
+   *
+   * <p>You must specify at least one orderBy clause for limitToLast queries, otherwise an exception
+   * will be thrown during execution.
+   *
+   * <p>Results for limitToLast queries cannot be streamed via the {@link
+   * #stream(ApiStreamObserver)} API.
+   *
+   * @param limit The maximum number of items to return.
```

Comment:
nit: the (no caps) and no period at end since this is not a complete sentence

Reply:
I left it as is for now to be consistent with the rest of the code. I will send a PR once this is merged to clean this up everywhere.

Reply:
Google Java practices explicitly rule out consistency with existing code as a justification for violating style guidelines in newly added code:
 
https://engdoc.corp.google.com/eng/doc/devguide/java/style/index.md?cl=head#s8.5.2-newly-added-code

Reply:
Updated